pkg/controller/routetableassociation/routetableassociation_controller.go

```diff
@@ -175,12 +176,12 @@ func (r *ReconcileRouteTableAssociation) Reconcile(request reconcile.Request) (r
 				if aerr, ok := ierr.(awserr.Error); ok {
 					switch aerr.Code() {
 					default:
-						fmt.Println(aerr.Error())
+						//fmt.Println(aerr.Error())
```

Comment:
Why did we comment these out?

Reply:
I had received some feedback a while back on not having anything print to the screen- I had commented them out, figuring I'd setup logging when I go back and refactor. I did go ahead and change those to log the events in the resource for this pull request.